source/includes/inkling2-reference/_curriculums.html.md

```diff
@@ -0,0 +1,122 @@
+# Curriculums
+
+```inkling2--code
+concept MyConcept(input): OutputType {
+  curriculum {
+    source MySimulator
+
+    # Lessons specified here
+
+  }
+}
+```
+
+The `curriculum` statement is used within a [concept][2] to specify how the concept should be trained.
+
+The [lessons][1] defined within the curriculum can be used to create a staged teaching plan.
+
+> Curriculum Syntax
+
+```inkling2--syntax
+curriculum {
+  source <simulatorReference>
+
+  [lessonStatement]*
+}
+```   
+
+### Usage
+
+There can be only one curriculum per concept, and every concept must have a curriculum.
+
+Every curriculum must provide a `source` clause that specifies the data source for teaching the concept. Currently, only [simulators][3] are supported as data sources, so the `source` keyword must be followed by a simulator declaration or the name of a simulator declared at the global scope of the program.
+
+The `train` keyword indicates which concept this curriculum trains.
+
+### Breakout Example
+
+> Breakout Example
+
+```inkling2--code
+simulator BreakoutSimulator(action: PlayerMove, config: BreakoutConfig): GameState {
+}
+```
+
+The `simulator` declaration specifies the simulator name and the types of two input values (the action and configuration). It also specifies the type of the simulator's output (the simulator state). In this instance, the configuration type is `BreakoutConfig`. The lessons initialize the configuration type of `BreakoutConfig`.
+
+
+```inkling2--code
+type BreakoutConfig {
+  level: Number.UInt16,
+  paddle_width: Number.UInt8<1 .. 4>,
+  bricks_percent: number
+}
+
+type PlayerMove number<Left = -1, Stay = 0, Right = 1>
+
+type GameState {
+  pixels: Image.Gray<84, 336>
+}
+
+graph (iput: GameState) {
+  concept HighScore(input): PlayerMove {
+    curriculum {
+      source BreakoutSimulator
+
+      lesson ScoreLesson {
+        constraint {
+          level: Number.UInt16<1 .. 100>,
+          paddle_width: Number.UInt8<1 .. 4>,
+          bricks_percent: 1
+        }
+      }
+    }
+  }
+
+  output PlayBreakout
+}
+```
+
+The configuration type `BreakoutConfig` is constrained in the lesson. When a concept is being trained, a new simulation configuration is passed to the simulator for each new episode. The configuration values are chosen in accordance with the current lesson's constraint. For example, in the lesson above, the `bricks_present` field is constrained to a value of 1, so the configuration passed to the simulator at the beginning of every episode will contain a value of 1 for this field. The `paddle_width` field is constrained to integers in the range of 1 through 4, so the configuration passed to the simulator at the beginning of each episode will contain a value of 1, 2, 3 or 4 for this field.
```

Comment:
should specify that values are chosen uniformly at randomly from the set specified in the constraint.